lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -33,12 +34,34 @@ def create_inference_job(
     return job_response
 
 
-@router.post("/evaluate/", status_code=status.HTTP_201_CREATED)
-def create_evaluation_job(
+@router.post("/annotate/", status_code=status.HTTP_201_CREATED)
+def create_annotation_job(
     service: JobServiceDep,
-    job_create_request: JobEvalCreate,
+    job_create_request: JobAnnotateCreate,
     request: Request,
-    response: Response
+    response: Response,
+) -> JobResponse:
+    # Lumigator's opinion on the best summarization model
+    # and the one that should generate annotations.
+    # In the future, we could expose this via a config file
+    # so that, for a supported task, there is a default
+    # "reference" model. For now, we keep the current functionality:
+    # Lumigator decides who annotates.
+
+    inference_job_create_request = JobInferenceCreate(
+        **job_create_request.dict(), model="hf://facebook/bart-large-cnn"
+    )
+    job_response = service.create_job(inference_job_create_request)
+
+    url = request.url_for(get_job.__name__, job_id=job_response.id)
+    response.headers[HttpHeaders.LOCATION] = f"{url}"
+
+    return job_response
```

Comment:
will the current generic job response include all the additional fields we need for inference as well, if there are any?